src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingService.cs

```diff
@@ -83,8 +83,10 @@ public async Task<TextEdit[]> FormatAsync(
         var filteredEdits = range is null
             ? result.Edits
             : result.Edits.Where(e => range.LineOverlapsWith(e.Range));
+        var minimalEdits = GetMinimalEdits(originalText, filteredEdits);
+        minimalEdits = NormalizeLineEndings(originalText, minimalEdits);
```

Comment:
I think the normalization should probably happen first, since it could affect what the "minimal" result is.